browser/.beforeprettier

```diff
@@ -75,6 +75,7 @@
 /src/control/Parts.js
 /src/control/Permission.js
 /src/control/Ruler.js
+/src/control/VRuler.ts
```

Comment:
please not add new files to ignore list, we want new formatting to be applied

Reply:
okay

Reply:
i see now code is formatted, I will push that change